hail/python/hail/methods/qc.py

```diff
@@ -492,13 +512,548 @@ def n_discordant(counter):
+                                                     intron=hl.tstr,
+                                                     lof=hl.tstr,
+                                                     lof_flags=hl.tstr,
+                                                     lof_filter=hl.tstr,
+                                                     lof_info=hl.tstr,
+                                                     minimised=hl.tint32,
+                                                     polyphen_prediction=hl.tstr,
+                                                     polyphen_score=hl.tfloat,
+                                                     protein_end=hl.tint32,
+                                                     protein_start=hl.tint32,
+                                                     protein_id=hl.tstr,
+                                                     sift_prediction=hl.tstr,
+                                                     sift_score=hl.tfloat,
+                                                     strand=hl.tint32,
+                                                     swissprot=hl.tstr,
+                                                     transcript_id=hl.tstr,
+                                                     trembl=hl.tstr,
+                                                     uniparc=hl.tstr,
+                                                     variant_allele=hl.tstr)),
+        variant_class=hl.tstr)
+
+    @staticmethod
+    def from_dict(config: Dict[str, Any]):
+        return HailVEPConfig(
+            data_bucket=config['data_bucket'],
+            data_mount=config['data_mount'],
+            image=config['image'],
+            regions=config['regions'],
+            cloud=config.get('cloud'),
+            reference_data_is_bucket_requester_pays=config['reference_data_is_bucket_requester_pays'],
+        )
+
+    def __init__(self,
+                 *,
+                 data_bucket: str,
+                 data_mount: str,
+                 image: str,
+                 regions: List[str],
+                 cloud: Optional[str],
+                 reference_data_is_bucket_requester_pays: bool):
+        self.data_bucket = data_bucket
+        self.data_mount = data_mount
+        self.image = image
+        self.regions = regions
+        self.env = {}
+        self.reference_data_is_bucket_requester_pays = reference_data_is_bucket_requester_pays
+        self.cloud = cloud
+        self.batch_run_command = ['python3', '/hail-vep/vep.py', 'vep']
+        self.batch_run_csq_header_command = ['python3', '/hail-vep/vep.py', 'csq_header']
+
+    @abc.abstractmethod
+    def vep_command(self,
+                    consequence: bool,
+                    tolerate_parse_error: bool,
+                    part_id: int,
+                    input_file: Optional[str],
+                    output_file: str) -> List[str]:
+        raise NotImplementedError
+
+
+class HailVEPConfig_GRCh37_85(HailVEPConfig):
```

Comment:
We're in the Hail library, I don't think we need to prefix this with `Hail`. Let's call it `VEPConfigGRCh37Version85`